sdk/resources/mgmt/src/main/java/com/azure/management/resources/fluentcore/arm/ExpandableStringEnum.java

```diff
@@ -15,36 +16,34 @@
  * @param <T> a specific expandable enum type
  */
 public abstract class ExpandableStringEnum<T extends ExpandableStringEnum<T>> {
```

Comment:
There should be a same class in core. May be we can remove this later.

Reply:
removed